eras/babbage/formal-spec/introduction.tex

```diff
@@ -23,5 +21,8 @@ \section{Introduction}
 \item Add inline datums in the UTxO
 \item Add reference scripts
 \item Add transaction fields for specifying and returning collateral
-\item Remove the protocol parameter $\var{d}$ and the overlay schedule
+\item Remove the protocol parameters $\var{d}$ and $\var{extraEntropy}$
```

Comment:
@WhatisRT just letting you know I snuck this in as well (mentioning removing the extra entropy).